targets/python/source/source/PlayFabHTTP.py

```diff
@@ -0,0 +1,36 @@
+import json
+import PlayFabSettings
+import PlayFabErrors
+import requests
+
+def DoPost(urlPath, request, authType, authKey, extraHeaders):
+    if PlayFabSettings.TitleId == None:
+        raise Exception("You must set your titleId before making an api call")
+
+    url = PlayFabSettings.GetURL(urlPath)
+
+    try:
+        j = json.dumps(request)
+    except:
+        err = PlayFabJsonError(request)
```

Comment:
Throw a PlayFabException for this case, saying the request isn't serializable, and reference the caught exception.